spec/00-abstract.md

```diff
@@ -0,0 +1,6 @@
+## Abstract
```

Comment:
no need. Header should be added by the reSpec

Reply:
Dropped it.